auth-header.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (req, res, next) => {
+    if (req.path.indexOf("/auth-header-required") === 0) {
+        if (req.get("Authorization") !== "Bearer 123456") {
```

Comment:
wait we're not checking this in const time! :wink:

Reply:
not again :wink: